src/Mod/Draft/draftguitools/gui_edit.py

```diff
@@ -313,6 +315,18 @@ def proceed(self):
         # self.alignWorkingPlane()
 
 
+    def numericInput(self, v, numy=None, numz=None):
+        """Execute callback by the toolbar to activate the update function.
+
+        This function gets called by the toolbar
+        or by the mouse click and activate the update function.
+        """
+        if numy:
+            v = App.Vector(v, numy, numz)
+        self.endEditing(self.obj, self.editing, v)
+        App.ActiveDocument.recompute()
```

Comment:
I haven't looked at the logic of this Draft Edit command with detail, but I wouldn't call the recompute in any function but the last one, that is, the finish method. But that's up to you.

Reply:
Sorry to respond so late... 
This has always been the behavior of Draft_Edit... I think it's good to recompute the document after an EditTracker is moved: the object is updated so it's good to see how this reflects on other objects too... Perhaps in the future a "skip recompute until the end" option can be added.